bin/record-build-info

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
```

Comment:
This isn't how the template app does this any more so depends what you want to do in that respect, or update at a later date.

Reply:
should I remove this bit? It is in the orginal send-legl-mail-to-prisons app so I copied it over. However am unsure as to why its needed

Reply:
I couldn't find any reference to it in the code so I reckon you could get rid of it (and the corresponding line added to `package.json`).

It was removed from the template app here (https://github.com/ministryofjustice/hmpps-template-typescript/pull/190) - I'm assuming this repo was cloned from the template after this.